rdr_service/genomic/genomic_job_components.py

```diff
@@ -257,7 +259,7 @@ def _ingest_genomic_file(self, file_obj):
         """
         Reads a file object from bucket and inserts into DB
         :param: file_obj: A genomic file object
-        :return: A GenomicSubProcessResultCode
+        :return: A GenomicSubProcessRes3ultCode
```

Comment:
Another typo?

Reply:
Ugh https://github.com/all-of-us/raw-data-repository/pull/2900/commits/d63ee306a76fa9ab1045b541e369ca3ba66a8ff5